backend/flask_api.py

```diff
@@ -31,34 +31,33 @@
 
 es = Elasticsearch([{'host': 'elasticsearch', 'port': 9200}])
 client = MongoClient('mongodb:27017')
-db = client.knexDB
+db = client.knexdb
 coll = db.projects
-app = Flask(__name__)
-CORS(app)
 
-with app.open_resource("manifest_schema.json") as schema_file:
+with open("manifest_schema.json") as schema_file:
```

Comment:
we should revert this to app.open_resource() and see what the problem with str/bytes was